nestkernel/connection_manager.cpp

```diff
@@ -1311,11 +1336,48 @@ nest::ConnectionManager::get_targets( std::vector< index > sources,
     target_it = targets.begin();
     for ( ; source_it != sources.end(); source_it++, target_it++ )
     {
-      if ( ( *it ).get( *source_it ) != 0 )
+      if ( validate_source_entry_( thread_id, *source_it ) != 0 )
       {
-        validate_pointer( ( *it ).get( *source_it ) )
+        validate_pointer( validate_source_entry_( thread_id, *source_it ) )
           ->get_target_gids( ( *target_it ), thread_id, synapse_model );
       }
     }
   }
 }
+
+void
+nest::ConnectionManager::get_targets( std::vector< index > sources,
```

Comment:
this function replaces the original `get_targets` function, correct? if so, you should remove the outdated function.